packages/webapp/src/components/lists/ReportList/useCsvExport.ts

```diff
@@ -25,6 +25,7 @@ export function useCsvExport(data: unknown[]) {
 	)
 	return {
 		downloadCSV,
-		setCsvFields
+		setCsvFields,
+		csvFields
```

Comment:
If there is a better way to access the `csvFields` in `ReportList` other than exporting them here, that would be preferable

Reply:
Nope, this is where the state is, otherwise you are going to pass it around 4~5 files like they did with `setCsvfields`. Which on itself is a nightmare.